content/api/vision/coordinates.md

```diff
@@ -47,9 +47,22 @@ robot, exactly 1 metre away.
 
 {{% added_in update="2018-jan" feature="spherical coordinates" %}}
 
-## Diagram
+### Diagram
 
 The following diagram shows the orientation of the Cartesian axes relative to
 the camera as well as the angles which describe the spherical coordinate space.
 
 ![A diagram showing the coordinate spaces](/img/api/coordinate-spaces.svg)
+
+## Orientation Coordinates
+
+Orientation represents the rotation of a marker around the x, y, and z axes.
```

Comment:
Any reason why X, Y, Z are capitalised and x,y,z are not?

Reply:
Nope, good catch